hrjobcontract/js/src/job-contract/controllers/modal/modal-contract.html

```diff
@@ -754,7 +754,7 @@ <h4>Life Insurance</h4>
             <div class="col-xs-12">
               <div class="form-group">
                 <label class="col-sm-4 control-label">Provider</label>
-                <div class="col-sm-6">
+                <div class="col-sm-5">
```

Comment:
What is the reason why we shrink it? Does it just look better as per your opinion?

Reply:
Yes, it looks much better aligning it to the plan type.